### PR TITLE
Add guide on how to use env variables with the CLI

### PR DESCRIPTION
**Pull request by silentworks**
## What kind of change does this PR introduce?

docs update

## What is the current behavior?

We don't have a guide on using environment variables with local Supabase

## What is the new behavior?

Added guide on how to use environment variables with local Supabase

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'using-envitonment-variables-in-your-config',
+  title: 'Using environment variables in your config.toml',
+  description: 'How to use environment variables in your config.toml with the Supabase CLI.'
+}
+
+The Supabase CLI can use environment variables from your `.env` file in your project's root directory inside 
+your `config.toml` file.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx:10]:
```suggestion
The Supabase CLI can use environment variables from your `.env` file in your project's root directory inside your `config.toml` file.
```

Could you please disable line-breaks for markdown files? It makes it harder to make suggestions in PRs

VSCode (and other editors) usually have a "wrap" configuration item which helps with readability on your local machine

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 2]
```diff
@@ -142,6 +142,10 @@ export const cli = {
   items: [
     { name: 'Overview', url: '/guides/cli' },
     { name: 'Managing Environments', url: '/guides/cli/managing-environments' },
+    {
+      name: 'Using environment variables in your config',
+      url: '/guides/cli/using-envitonment-variables-in-your-config',
```
Comment (kiwicopple) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:147]:
```suggestion
      name: 'Using environment variables in config.toml',
      url: '/guides/cli/using-envitonment-variables-in-config',
```

I suggest we avoid words like "your" in page titles

File: apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx (hunk 1 above)
Comment (silentworks) [reply apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx:10]: Will do.

File: apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'using-envitonment-variables-in-your-config',
```
Comment (gregnr) [inline apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx:4]:
```suggestion
  id: 'using-environment-variables-in-your-config',
```

Heads up the name of this `mdx` file also has this typo.

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 4]
```diff
@@ -142,6 +142,10 @@ export const cli = {
   items: [
     { name: 'Overview', url: '/guides/cli' },
     { name: 'Managing Environments', url: '/guides/cli/managing-environments' },
+    {
+      name: 'Using environment variables in config.toml',
+      url: '/guides/cli/using-envitonment-variables-in-config',
```
Comment (gregnr) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:147]:
```suggestion
      url: '/guides/cli/using-environment-variables-in-config',
```

**Review by gregnr — APPROVED** [review]

**Pull request merged**